src/main/java/roomescape/payment/domain/Payment.java

```diff
@@ -0,0 +1,60 @@
+package roomescape.payment.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import roomescape.member.domain.Member;
+import roomescape.reservation.domain.Reservation;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Payment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @JoinColumn(nullable = false)
+    @ManyToOne(optional = false, fetch = FetchType.LAZY)
+    private Member member;
```

Comment:
```suggestion
    @JoinColumn(name = "member_id", nullable = false)
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    private Member member;
```
컬럼명을 명시적으로 지정해주면 좋을 것 같아요

Reply:
피드백 주셔서 감사합니다. 
`JoinColumn`에 `name`을 종종 작성하시는 분을 보고 어차피 자동으로 만들어주는데 왜 다는거지" 라고 생각하며 사소하다고 느꼈기에 그동안 신경쓰지 않았는데 덕분에 공부하게 되었습니다. 

name을 사용하지 않으면 JPA는 암시적 네이밍 전략을 통해 조인 컬럼의 기본 이름을 유추 합니다.

위 상황을 예를 들면  
>조인 대상 필드명 (member) + "_" + 참조 엔티티의 기본 키 컬럼명 (id)
```bash
"member" + "_" + "id" = "member_id"
```

하지만 만약 아래와 같이 참조 엔티티의 PK가 `@Column(name="member_id") private Long id;` 처럼 커스텀 이름을 갖고 있으면 기존과 다르게 생성됩니다.

Reply:
잘 정리해주셨네요 💯